Assets/AnkrSDK/Examples/UseCases/Ads/AdsCallbackListener.cs

```diff
@@ -0,0 +1,171 @@
+using AnkrSDK.Ads.UI;
+using Cysharp.Threading.Tasks;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace AnkrSDK.UseCases.Ads
+{
+	public class AdsCallbackListener : MonoBehaviour
+	{
+		private const string FullscreenAdTestUnitId = "2592a670-f7ab-4400-ab95-af7d5975a9f8";
```

Comment:
Remove duplicate const and add it into one file where both monobehs are using it